insights/combiners/tests/test_httpd_conf.py

```diff
@@ -178,28 +178,36 @@ def test_active_httpd_nest():
-    #             ('deny,allow', 'Order deny,allow', 'FilesMatch', '".php[45]?$"', 'httpd.conf', '/etc/httpd/conf/httpd.conf')]}]
-    # assert result.get_setting_list('RewriteEngine', 'IfModule') == [
-    #         {('IfModule', 'mod_rewrite.c'): [
-    #             ('On', 'RewriteEngine On', 'IfModule', 'mod_rewrite.c', 'httpd.conf', '/etc/httpd/conf/httpd.conf'),
-    #             ('Off', 'RewriteEngine Off', 'IfModule', 'mod_rewrite.c', 'httpd.conf', '/etc/httpd/conf/httpd.conf')]},
-    #         {('IfModule', 'mod_rewrite.c'): [
-    #             ('Off', 'RewriteEngine Off', 'IfModule', 'mod_rewrite.c', 'httpd.conf', '/etc/httpd/conf/httpd.conf'),
-    #             ('On', 'RewriteEngine On', 'IfModule', 'mod_rewrite.c', '00-z.conf', '/etc/httpd/conf.d/00-z.conf')]}
-    # ]
-    # assert result.get_setting_list('EnableSendfile') == [
-    #         ('off', 'EnableSendfile off', None, None, '00-z.conf', '/etc/httpd/conf.d/00-z.conf'),
-    #         ('on', 'EnableSendfile on', None, None, 'httpd.conf', '/etc/httpd/conf/httpd.conf')]
+    php_fm_order = result.get_setting_list('Order', section=('FilesMatch', 'php'))
+    assert {
+            ('FilesMatch', '".php[45]?$"'): [
+                ('allow,deny', 'Order allow,deny', 'FilesMatch', '".php[45]?$"', '00-z.conf', '/etc/httpd/conf.d/00-z.conf'),
+                ('deny,allow', 'Order deny,allow', 'FilesMatch', '".php[45]?$"', '00-z.conf', '/etc/httpd/conf.d/00-z.conf')]
+           } in php_fm_order
+    assert {
+            ('FilesMatch', '".php[45]"'): [
+                ('allow,deny', 'Order allow,deny', 'FilesMatch', '".php[45]"', 'httpd.conf', '/etc/httpd/conf/httpd.conf'),
+                ('deny,allow', 'Order deny,allow', 'FilesMatch', '".php[45]"', '00-z.conf', '/etc/httpd/conf.d/00-z.conf')],
+           } in php_fm_order
+    assert {
+            ('FilesMatch', '".php[45]?$"'): [
+                ('allow,deny', 'Order allow,deny', 'FilesMatch', '".php[45]?$"', 'httpd.conf', '/etc/httpd/conf/httpd.conf'),
+                ('deny,allow', 'Order deny,allow', 'FilesMatch', '".php[45]?$"', 'httpd.conf', '/etc/httpd/conf/httpd.conf')]
+           } in php_fm_order
+    re_im = result.get_setting_list('RewriteEngine', 'IfModule')
+    assert {
+            ('IfModule', 'mod_rewrite.c'): [
+                ('On', 'RewriteEngine On', 'IfModule', 'mod_rewrite.c', 'httpd.conf', '/etc/httpd/conf/httpd.conf'),
+                ('Off', 'RewriteEngine Off', 'IfModule', 'mod_rewrite.c', 'httpd.conf', '/etc/httpd/conf/httpd.conf')]
+           } in re_im
+    assert {
+            ('IfModule', 'mod_rewrite.c'): [
+                ('Off', 'RewriteEngine Off', 'IfModule', 'mod_rewrite.c', 'httpd.conf', '/etc/httpd/conf/httpd.conf'),
+                ('On', 'RewriteEngine On', 'IfModule', 'mod_rewrite.c', '00-z.conf', '/etc/httpd/conf.d/00-z.conf')]
+           } in re_im
+    sorted(result.get_setting_list('EnableSendfile')) == sorted([
```

Comment:
@xiangce are you missing an assert here and on line 229 below?

Reply:
My bad, thank you @bfahr.